### ISSUE TITLE
Execute on mouse hovering

### ISSUE THREAD

**Issue by Kochise** [bitagoras/PyPadPlusPlus#3]
Hi, despite `popupForSelectedExpression = False` if `pyPadStart.py`, once the kernel starts evaluating (with either `pyPadExecute.py` or `pyPadRestart.py` getting triggered), everything gets evaluated as soon as the mouse hover it, **even not moving it** cycle evaluation about once or twice a second (not regular timing).

Was expecting more the tooltip to appears instead.

I've tried seeting `CT_expression = False` in `__init__.py:636` (which I guess should be the correct default value, regarding the comment in `pyPadStart.py`) but to no avail.

Any guess ? Something wrong in `def onMouseDwell(self, args)` that triggers the evaluation upon mouse position ?

Regards.

**Comment by bitagoras** [comment]
Can you give me an example to better describe your problem? What do you mean by "gets evaluated"? Is the whole line executed?
The `CT_expression = True` should remain, otherwise the evaluation of an unselected variable below the curser will not work any more by the keyboard shortcut (e.g. shift-ctrl-space).

**Comment by Kochise** [comment]
Basically, when opening the `pyPadDemo_py3.py` file, clicking on the `1+2` statement (line 14), pressing `Shift+Enter` you get the expected `3` result printed in the console, but over and over as your mouse cursor stays over it, and when you point on another line with the mouse, **not even selecting**, the said line gets executed as well, over and over.

**Comment by bitagoras** [comment]
This look like a problem of the mouse middle button which always executes the line. You can disable it by commenting out the line
`self.onTimerMiddleButton()  # start periodic timer to check state of middleButton`
Do you use the 64bit version of Notepadd++? There is a problem with 64 bit and the mouse middle button.

**Comment by Kochise** [comment]
No, I always use the 32 bit version, I'll try the solution you provided and give an update.

**Comment by Kochise** [comment]
Seems like it comes from my Logitech G600 middle button that gave me a middle finger.
